### PR TITLE
Lesson 1.4 assignment is done

### PR DESCRIPTION
**Pull request by omer-jan**
lesson 1.4 assignment is done.

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -1,17 +1,33 @@
-import React from "react"; 
+import React, { useState } from 'react';
 let AddTodoForm=(props)=>{
```
Comment (kwilson05) [inline src/AddTodoForm.js:2]: Try deconstructing the properties you need from props here

File: src/AddTodoForm.js [hunk 2]
```diff
@@ -1,17 +1,33 @@
...
+// State
+const [todoTitle, setTodoTitle] = useState();
+
+// Destructing props
+const{onAddTodo}=props;
+
+let handleTitleChange=(e)=>{
+     
+     let newTodoTitle=e.target.value;  
+     setTodoTitle(newTodoTitle);
```
Comment (kwilson05) [inline src/AddTodoForm.js:13]: nice!!! 💯

File: src/App.js [hunk 3]
```diff
@@ -2,15 +2,20 @@ import React, { useState } from 'react';
...
 import TodoList from "./TodoList";
 
-function App() {
-  const [newTodo, setNewTodo] = useState("");
+function App() { 
+
+  const [todoList, setTodoList] = useState([]);
+
+  let addTodo =(newTodo)=>{ 
+    setTodoList([...todoList,newTodo])
```
Comment (kwilson05) [inline src/App.js:10]: 💯

File: src/TodoList.js [hunk 4]
```diff
@@ -1,28 +1,11 @@
...
-      id:4,
-      title:"Prepare for the next session"
-    },
-    
-  ]
-let TodoList=()=>{
+let TodoList=(props)=>{ 
+
+  // Destructing  props
+  const {todoList}=props;
```
Comment (kwilson05) [inline src/TodoList.js:6]: There go!! Maybe deconstruct the props in the function's signature next time :)

File: src/TodoListItem.js [hunk 5]
```diff
@@ -1,3 +1,10 @@
 import React from 'react'
-let TodoListItem=(props)=><li key={props.toDo.id}>{props.toDo.title}</li>  
+let TodoListItem=(props)=>{
```
Comment (kwilson05) [inline src/TodoListItem.js:2]: Same thing here! Deconstruct in the signature like 
```suggestion
let TodoListItem=({toDo})=>{
```

**Review by kwilson05 — APPROVED** [review]
Good job, Mohammad !!! I want you to try and use `const` instead of `let` more often.
Also, try deconstructing the props in a function's signature instead within the function's body.

Otherwise, keep up the good work

File: src/TodoListItem.js (hunk 5 above)
Comment (omer-jan) [reply src/TodoListItem.js:2]: Got it. thank you

**Pull request merged**